samples/framework/libraries/migrate-library-csproj/src/Car/Car.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net45;net451;netstandard1.6</TargetFrameworks>
```

Comment:
Seems unusual to target `net45` and `net451`. They're supported TFM's, but they're unsupported frameworks. Can it be `net452;netstandard1.6`?